src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/oppgave/OppgaveDomain.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.tilleggsstonader.sak.opplysninger.oppgave
+
+import no.nav.tilleggsstonader.kontrakter.oppgave.Oppgavetype
+import no.nav.tilleggsstonader.sak.infrastruktur.database.Sporbar
+import org.springframework.data.annotation.Id
+import org.springframework.data.relational.core.mapping.Embedded
+import org.springframework.data.relational.core.mapping.Table
+import java.util.UUID
+
+@Table("oppgave")
+data class OppgaveDomain(
+    @Id
+    val id: UUID = UUID.randomUUID(),
+    val behandlingId: UUID,
```

Comment:
Burde behandlingId være optional? 
Alt kan vi åpne opp for den senere :) (den er nullable i databasen)

Reply:
Kanskje 🤔  Den er ikke nullable i ef-sak, men tenker det kan være mulig å opprette oppgaver uten behandling.

Burde uansett være konsistent med domeneobjektet.

Reply:
Eller, tenker vi kan starte med å den ikke nullable. Lettere å angre på det.